packages/url/url.js

```diff
@@ -14,6 +14,7 @@ Window.prototype.forceJURL = false;
       var u = new URL('b', 'http://a');
       u.pathname = 'c%20d';
       hasWorkingUrl = u.href === 'http://a/c%20d';
+    // eslint-disable-next-line no-empty
     } catch(e) {}
```

Comment:
```suggestion
    }
```

(depends on transpiler)